src/Footer/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+import { Link } from 'react-router-dom'
+import './Footer.scss'
+
+// pass in buttons as props to generate links for footer
```

Comment:
Was this a note from Kaitlin?

Reply:
Yeah it is. She was suggesting we make separate components to render in Footer